### PR TITLE
contrib: Add Dockerfile.builder.fedora

### PR DESCRIPTION
**Pull request by dougsland**
The Fedora distro version of Dockerfile.builder

Signed-off-by: Douglas Schilling Landgraf <dougsland@gmail.com>

### CODE REVIEW

File: contrib/Dockerfile.builder.fedora [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		python3-devel \
+		python3-pip \
+		python3-setuptools \
+		net-tools \
+		libsodium-devel \
+		net-tools \
+		valgrind \
+		wget && \
+	dnf clean all && \
+	git clone https://github.com/ElementsProject/lightning.git && \
```
Comment (dougsland) [inline contrib/Dockerfile.builder.fedora]: @cdecker actually, I have added a git clone here for lightning.git which don't have in Dockerfile.builder. Should I remove it?

**Comment by dougsland** [comment]
Fell free to squash the commits and merge.

**Comment by cdecker** [comment]
Great work @dougsland, are you actively testing commits with this builder? Our travis-ci instance is already taking forever with the ubuntu-only matrix, maybe you have some more capacity?

ACK a8c3466

**Comment by dougsland** [comment]
@cdecker it's more like a local machine test env and devel env @ home. That's why I had the git clone && building of c-lightning as well but "I figure out" let's just mimic for now what we have in ubuntu for fedora and encourage more people to use it. Other question, what's the plan to have a Dockerfile to setup a devel setup? Might bring more people to the game.

**Comment by cdecker** [comment]
Not using docker images to develop currently, but when I did I would just bind mount the checked out source into the container and make sure that my user in the container matches the user outside.

**Comment by dougsland** [comment]
Similar what I do, just shared the devel approach for quick deploy as some people showed interested. Indeed, for tests over commit seems better approach.

**Pull request merged**